lib/streamlit/elements/widgets/multiselect.py

```diff
@@ -105,8 +107,8 @@ def _check_max_selections(
 
     default_count = _get_default_count(selections)
     if default_count > max_selections:
-        raise StreamlitAPIException(
-            _get_over_max_options_message(default_count, max_selections)
```

Comment:
nit: is `_get_over_max_options_message` still used or can it be deleted from codebase?